packages/grid/data-grid/src/tests/overrides.test.ts

```diff
@@ -0,0 +1,15 @@
+import { createTheme } from '@material-ui/core';
+
+createTheme({
+  overrides: {
```

Comment:
This is the API of v4, which we do no longer optimize for. Could we test the v5 API?

Reply:
The `components` ThemeOptions isn't available in `createTheme` on `@material-ui/core` under `packages/grid/data-grid`.  Following v5 api: https://next.material-ui.com/customization/theme-components/#global-style-overrides
Do we support v5 here?